tests/binary-simple/BUILD

```diff
@@ -14,6 +14,10 @@ haskell_import(
 haskell_test(
     name = "binary-simple",
     srcs = ["Main.hs"],
+    compiler_flags = [
+        "-threaded",
```

Comment:
May be worth having a test with something that only compiles with e.g. `-XOverloadedStrings` to ensure the flags are actually passed

Reply:
We already have a `binary-with-compiler-flags`, it just didn't have any whitespace. I could move those arguments there, actually